source/php-frameworks/drupal.txt

```diff
@@ -0,0 +1,110 @@
+.. _php-drupal-integration:
+
+==========================
+Drupal MongoDB Integration
+==========================
+
+.. facet::
+   :name: genre
+   :values: tutorial
+
+.. meta::
+   :keywords: php framework, CMS, web app, authentication, Atlas
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+Overview
+--------
+
+In this guide, you can learn about the benefits of using MongoDB Atlas
+as the database for Drupal sites and navigate to a tutorial that
+demonstrates how to set up the MongoDB driver for Drupal.
+
+`Drupal <https://www.drupal.org/about>`__ is an open-source web content
+management system (CMS) written in PHP. Drupal provides simple content
+authoring, composability, and robust authentication features.
+
+The :ref:`php-drupal-tutorial` section of this guide links to a tutorial
+which you can follow to set up a Drupal site that connects to MongoDB Atlas.
+
+The :ref:`php-drupal-resources` section contains links to resources and
+documentation for further learning.
+
+Why Use MongoDB as the Database for Drupal Sites?
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Websites that seek to provide personalized user experiences must contain
+robust capabilities for user authentication. Sites that have many
+authenticated users might face performance impacts due to the complexity
+of retrieving entity data from multiple tables in a relational database.
+MongoDB supports eliminates the need for complicated table joins and
```

Comment:
I: assume this should just be 'MongoDB eliminates'?

```suggestion
MongoDB eliminates the need for complicated table joins and
```

Reply:
yerp